utils/build-presets.ini

```
@@ -1423,6 +1423,9 @@ darwin-toolchain-version=%(darwin_toolchain_version)s
 darwin-toolchain-alias=%(darwin_toolchain_alias)s
 darwin-toolchain-require-use-os-runtime=0
 
+# Build the swiftmodule/tbd for armv7k watchos and i386 watchsimulator.
+swift-darwin-module-archs=armv7k i386
```

I suspect that for this to be ultimately considered [by CMake as a list](https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#lists) the values should be separated by a semicolon
```suggestion
swift-darwin-module-archs=armv7k;i386
```